packages/ui-components/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@eth-optimism/ui-components",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "build": "vite build && tsc -p tsconfig.json",
```

Comment:
something to consider in the future. i ended up using tsup https://github.com/ethereum-optimism/ecosystem/blob/1c11cff3566b6007828f759b0ce655b9055b184c/packages/op-app/package.json#L9 instead of just vite. felt like it gives more control and we could reuse it for backend packages

Reply:
nah don't worry about it right now, something we can talk about as a team to decide what we want to do. The main difference atleast from what i noticed is `tsup` gives more control over the `*.d.ts` files that get generated. Once we add more components we can see both ways work.